exec/java-exec/src/main/resources/rest/static/js/ace-code-editor/snippets/sql.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Drill SQL Syntax Snippets
+ */
+
+ace.define("ace/snippets/sql",["require","exports","module"], function(require, exports, module) {
+"use strict";
+
+exports.snippetText = "snippet info\n\
+	select * from INFORMATION_SCHEMA.${1:<tableName>};\n\
+snippet sysmem\n\
+	select * from sys.mem;\n\
+snippet sysopt\n\
+	select * from sys.opt;\n\
+snippet sysbit\n\
+	select * from sys.bit;\n\
+snippet sysconn\n\
+	select * from sys.conn;\n\
+snippet sysprof\n\
+	select * from sys.prof;\n\
+snippet cview\n\
```

Comment:
I am not sure that snippets for sys tables are correct. Please see below list of current tables present in sys schema:
```
0: jdbc:drill:drillbit=localhost> show tables;
+---------------+-----------------------+
| TABLE_SCHEMA  |      TABLE_NAME       |
+---------------+-----------------------+
| sys           | profiles_json         |
| sys           | drillbits             |
| sys           | boot                  |
| sys           | internal_options      |
| sys           | threads               |
| sys           | options_val           |
| sys           | profiles              |
| sys           | connections           |
| sys           | internal_options_val  |
| sys           | memory                |
| sys           | version               |
| sys           | options               |
```